pytest_watch/watcher.py

```diff
@@ -164,8 +173,11 @@ def watch(directories=[], ignore=[], auto_clear=False, beep_on_failure=True,
         observer.join()
     except KeyboardInterrupt:
         observer.stop()
-    if onexit:
-        os.system(onexit)
+        if onsigint:
+            os.system(onsigint)
+    else:
+        if onexit:
+            os.system(onexit)
```

Comment:
You're right probably, and I've been using the other interrupt handler then only (https://github.com/joeyespo/pytest-watch/pull/46/files#diff-298efc967591604e348416060d077075R129).
Feel free to fix / enhance.